recipes/caper/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "caper" %}
+{% set version = "1.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 33a963c47f9b43c6070c81cd80a40b27df3c00ccc1f7078045a84bb0a7ea8f59 
+
+build:
+  skip: True  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
```

For non-`noarch: python` packages we have to resort to the `[py<36]` selector for skipping the build.
```suggestion
  skip: True  # [py<36 or win]
  number: 0
  script: "{{ PYTHON }} -m pip install . -vv"

requirements:
  host:
    - python
    - pip
  run:
    - python
```